AppSyncRealTimeClient/ConnectionProvider/AppsyncRealtimeConnection/RealtimeConnectionProvider+StaleConnection.swift

```diff
@@ -36,13 +38,19 @@ extension RealtimeConnectionProvider {
             guard let self = self else {
                 return
             }
-            AppSyncLogger.debug("[RealtimeConnectionProvider] Status: \(self.status). Connectivity status: \(connectivity.status)")
+            AppSyncLogger.debug(
+                "[RealtimeConnectionProvider] Status: \(self.status). Connectivity status: \(connectivity.status)"
+            )
             if self.status == .connected && connectivity.status == .unsatisfied && !self.isStaleConnection {
```

Comment:
Not PR applicable, but it would be nice to pull these out into descriptive local computed variables or functions at some point.